host/types/types.go

```diff
@@ -275,7 +275,18 @@ type DiscoverdConfig struct {
 
 type HostStatus struct {
 	ID        string           `json:"id"`
+	PID       int              `json:"pid"`
 	URL       string           `json:"url"`
 	Discoverd *DiscoverdConfig `json:"discoverd,omitempty"`
 	Network   *NetworkConfig   `json:"network,omitempty"`
 }
+
+type Command struct {
+	Path string   `json:"path"`
+	Args []string `json:"args"`
+	PID  int      `json:"pid"`
+}
+
+type LogBuffers map[string]LogBuffer
+
+type LogBuffer map[string][]byte
```

Comment:
This will encode the data to base64 in the JOSN, perhaps it should be a `string`?